turbinia/lib/docker_manager.py

```diff
@@ -150,20 +150,27 @@ def _create_mount_points(self, mount_paths, mode='rw'):
         list: The device blocks that will be mounted.
         dict: The file paths that will be mounted.
     """
+    accepted_vars = ['rw', 'ro']
     device_paths = []
     file_paths = {}
 
-    for mpath in mount_paths:
-      if mpath not in file_paths.keys() and mpath not in device_paths:
-        if IsBlockDevice(mpath):
-          formatted_path = '{0:s}:{0:s}:{1:s}'.format(mpath, 'r')
-          device_paths.append(formatted_path)
-        else:
-          file_paths[mpath] = {'bind': mpath, 'mode': mode}
+    if mode in accepted_vars:
+      for mpath in mount_paths:
+        if mpath not in file_paths.keys() and mpath not in device_paths:
+          if IsBlockDevice(mpath):
+            formatted_path = '{0:s}:{0:s}:{1:s}'.format(mpath, 'r')
+            device_paths.append(formatted_path)
+          else:
+            file_paths[mpath] = {'bind': mpath, 'mode': mode}
+    else:
+      log.warning(
```

Comment:
Instead of just logging a warning here, should we maybe raise an exception?  If this triggers it would be done by other bad code, and not user input, so we'd probably want to know right away (and not run into it indirectly when debugging potential downstream problems caused by this not getting mounted).

Reply:
Yep agreed. Updated it to raise an exception instead.